application/models/bestitamazonpay4oxidclient.php

```diff
@@ -801,7 +804,7 @@ public function saveCapture($oOrder = null)
      * @return stdClass
      * @throws Exception
      */
-    public function refund($oOrder = null, $fPrice, $aRequestParameters = array())
+    public function refund($oOrder = null, $fPrice = 0.0, $aRequestParameters = array())
```

Comment:
Is this really a good idea? Should $oOrder = null and a price of 0.0 be allowed to refund? would cause an amazon pay api error.

Reply:
Thanks for the comment. Has already been fixed.